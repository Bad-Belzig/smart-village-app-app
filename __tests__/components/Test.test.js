import React from 'react';
import renderer from 'react-test-renderer';
import { Test } from '../Test';

test('renders correctly', () => {
  const tree = renderer.create(<Test />).toJSON();
  expect(tree).toMatchSnapshot();
});
