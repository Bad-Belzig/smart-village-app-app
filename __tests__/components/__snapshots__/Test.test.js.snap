// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    Welcome to React Native!
  </Text>
  <Text
    style={
      Object {
        "color": "#333333",
        "marginBottom": 5,
        "textAlign": "center",
      }
    }
  >
    This is a React Native snapshot test.
  </Text>
</View>
`;
